README.md

```diff
@@ -41,19 +41,27 @@ See the below table to find out which version of the template to use.
 
 #### React Native <=> Template Version
 
-| React Native  	| Template  	|
-|---	            |---	        |
-| 0.64  	        | 6.6.*       |
-| 0.63  	        | 6.5.*       |
-| 0.62  	        | 6.4.*       |
-| 0.61  	        | 6.3.*       |
-| 0.60  	        | 6.2.*       |
+| React Native | Template |
+| ------------ | -------- |
+| 0.64         | 6.6.\*   |
+| 0.63         | 6.5.\*   |
+| 0.62         | 6.4.\*   |
+| 0.61         | 6.3.\*   |
+| 0.60         | 6.2.\*   |
 
-### Note on the legacy CLI
-There seems to be quite some confusion about the legacy CLI. This template only works with the new CLI. Make sure you have uninstalled the legacy `react-native-cli` first (`npm uninstall -g react-native-cli`), for the below command to work. If you wish to not use `npx`, you can also install the new CLI globally (`npm i -g @react-native-community/cli` or `yarn global add @react-native-community/cli`).
+## :warning: React Native CLI
 
-Further information can be found here: https://github.com/react-native-community/cli#about
+This template only works with the new CLI. Make sure you have uninstalled the legacy `react-native-cli` first (`npm uninstall -g react-native-cli`), for the below command to work. If you wish to not use `npx`, you can also install the new CLI globally (`npm i -g @react-native-community/cli` or `yarn global add @react-native-community/cli`).
+
+### Using this template with the old CLI
```

Comment:
This reads like it can be used with the old CLI, which is not true. The `--ignore-existing` flag forces `npx` to use the latest version of the CLI by ignoring what is installed locally.

I would propose to change it to:

If you tried the above and still get the `react-native-template-react-
native-template-typescript: Not found` error, please try adding the `--ignore-existing` flag to the `npx` call to force `npx` to ignore any locally installed versions of the CLI.

Reply:
Yeah that makes sense :)